### PR TITLE
Update our documentation

### PR DESCRIPTION
**Pull request by sgibson91**
This PR refreshes our documentation to include all the changes that happened in #1027

### CODE REVIEW

File: docs/howto/operate/new-cluster/new-cluster.md [hunk 1]
```diff
@@ -206,17 +206,17 @@ Then, output the credentials created by terraform to a file under the `secrets`
...
 
 ````{tabbed} Azure
 ```bash
-terraform output -raw kubeconfig > ../../secrets/CLUSTER_NAME.yaml
+terraform output -raw kubeconfig > ../../config/clusters/<cluster_name>/deployer-credentials.secret.yaml
 ```
 ````
 
-where `CLUSTER_NAME` matches the name of our `.tfvars` file.
+where `<cluster_name>` matches the name of our `.tfvars` file.
```
Comment (sgibson91) [inline docs/howto/operate/new-cluster/new-cluster.md:219]:
```suggestion
```

File: docs/howto/operate/new-cluster/new-cluster.md [hunk 2]
```diff
@@ -206,17 +206,16 @@ Then, output the credentials created by terraform to a file under the `secrets`
 
 ````{tabbed} Google Cloud
 ```bash
-terraform output -raw ci_deployer_key > ../../secrets/CLUSTER_NAME.json
+terraform output -raw ci_deployer_key > ../../config/cluster/<cluster_name>/deployer-credentials.secret.json
```
Comment (consideRatio) [inline docs/howto/operate/new-cluster/new-cluster.md:209]:
```suggestion
terraform output -raw ci_deployer_key > ../../config/clusters/<cluster_name>/deployer-credentials.secret.json
```

**Review by consideRatio — APPROVED** [review]
Looks amazing! Just a singly typo detected among all this!

EDIT: while I managed to produce a typo in my single line comment ^^

**Review by GeorgianaElena — APPROVED** [review]
:rocket:

**Pull request merged**